test/unit/sthDatabase_test.js

```diff
@@ -1192,6 +1205,12 @@ describe('sthDatabase tests', function() {
 
             describe('access', collectionAccessTests);
         });
+        
+        describe('collection access', function() {
```

Comment:
`'collection access'` seems to be copy-pasted from above line L1201. Please use a more meaninfull description (eg. "database not connected").